app/controllers/pseuds_controller.rb

```diff
@@ -146,4 +146,14 @@ def destroy
 
     redirect_to(user_pseuds_url(@user))
   end
+
+  private
+
+  def pseud_params
+    params.require(:pseud).permit(
+      :user_id, :name, :description, :is_default, :icon, :delete_icon, :icon_file_name,
```

Comment:
It does seem to work without `:icon_alt_text`, both in the sense that the pseud is created/updated *and* the specified alt text is saved, but I'm a little confused _why_ it works. Do you know, james? (I checked and taking out something like `:name` does cause a problem.)